changelog.md

```diff
@@ -24,10 +24,12 @@
   * `function: TransformerFunctionKind`
 
 ##### Enhancements
-* None.
+* Enable asyncio for gRPC from the newest update of grpcio and grpcio-tool
 
 ##### Fixes
 * Updated gRPC to fix support for latest LetsEncrypt certificates.
+* By pass grpc.channel_ready_future() for async io path.
+* Ensure insecure channel is used on async io path when secure is set to False.
```

Comment:
No need for these two lines as they are fixes for the current feature 'Enable asyncio for gRPC from the newest update of grpcio and grpcio-tool'.